engine/seq/sequencer.py

```diff
@@ -59,10 +60,15 @@ def _get_deltatime(self) -> float:
         return delta
 
     def _update(self) -> None:
+        time.sleep(0.008333333)
```

Comment:
Is this one needed? There will be some sleep due to the gui polling/event handling. Either way, should probably not be a magic number.

Reply:
this is "basically" 120fps, without it, python burns a core to the ground, and my gpu is rendering half a trillion frames 
glad to change this.